pages/status.tsx

```diff
@@ -93,14 +99,16 @@ const Status: FC = () => {
       {}
       {checkStatusReponse !== undefined ? (
         <StatusInfo
-          handleGoBackClick={handleGoBack}
-          goBackText={t('go-back')}
+          handleGoBackClick={
+            checkStatusReponse ? () => Router.push('/landing') : handleGoBack
```

Comment:
I think moving this if condition inside `handleGoBack` and set `onGoBackClick` prop with `handleGoBack` as before would make more sense.

We should also rename `handleGoBackClick` to `onGoBackClick` to be consistent with other event property naming convention, as example `onClick`, `onChange`, etc...

Reply:
It's why I was using 2 different status component before. 1 for when you receive an actual status `StatusInfo` and one when status is null `StatusInfoUnavailable`. It made the code easier to follow.

Reply:
If your suggesting it goes back to a single condition now, I'm okay to do that.

Reply:
For now, yes.